src/sentry/integrations/slack/tasks.py

```diff
@@ -318,3 +323,16 @@ def link_slack_user_identities(integration: Integration, organization: Organizat
                     "type": idp.type,
                 },
             )
+
+
+# TODO: add retry logic
+@instrumented_task(name="sentry.integrations.slack.post_message", queue="integrations", max_retries=0)  # type: ignore
```

Comment:
Do you also want late_acks? That will ensure that your messages are retried if a worker is restarted in while this task is running.

Reply:
@markstory At this point, this task just exists so we can make it async. We'll have to follow up later with handling failures. That's why I set `max_retries` to 0.